test/net/sourceforge/kolmafia/request/UseItemRequestTest.java

```diff
@@ -302,6 +306,56 @@ void maxUsesWorksForHtmlSpleenItemsWithConsumptionType(int spleenUsed, int maxUs
     }
   }
 
+  @Nested
+  class zeroFullness {
+    @ParameterizedTest
+    @CsvSource({"5140", "10883"})
+    void itShouldNotDivideByZeroWhenConsumingAstralEnergyDrink(int itemID) {
+      int spleenUsed = 0;
+      int maxUses = Integer.MAX_VALUE;
+      var cleanups =
+          new Cleanups(
+              withSpleenUse(spleenUsed),
+              withHP(0, 10, 10),
+              withMP(0, 10, 10),
+              withPath(AscensionPath.Path.STANDARD));
+
+      try (cleanups) {
+        assertThat(UseItemRequest.maximumUses(itemID, ConsumptionType.SPLEEN), is(maxUses));
+      }
+    }
+    /**
+     * The list ot tested items was derived from items with zero fullness in fullness.txt at the
+     * time the test was written. Magical sausage 10060 has zero fullness but it is not tested here
+     * because the fullness is controlled by a preference. Similarly for the glitch season reward
+     * 10207. Given that the point of this test is to show a divide by zero does not occur, leaving
+     * them out is deemed acceptable.
+     */
+    @ParameterizedTest
+    @CsvSource({"572", "1266", "1432", "2149", "2188", "4412"})
+    void itShouldNotDivideByZeroWhenConsumingZeroFullnessItems(int itemID) {
+      int fullness = 0;
+      int maxUses = Integer.MAX_VALUE;
+      var cleanups = withFullness(fullness);
+
+      try (cleanups) {
+        assertThat(UseItemRequest.maximumUses(itemID, ConsumptionType.EAT), is(maxUses));
+      }
+    }
+
+    @ParameterizedTest
+    @CsvSource({"1047", "1276", "4413", "7491"})
+    void itShouldNotDivideByZeroWhenConsumingZeroInebrietyItems(int itemID) {
```

Comment:
I would say you only need 1 zero drunk item to prove it works. The logic is the same for all of them.

Reply:
I was responding to a suggestion by @BadHorseMonkey at KoLmafia.us.  If I misunderstood or it is no longer a good idea then I will gladly reduce the test to one case,